examples/bulma.rs

```diff
@@ -2,21 +2,17 @@ use dioxus::prelude::*;
 use dioxus_markdown::Markdown;
 
 fn main() {
-    dioxus_desktop::launch(app);
+    dioxus::launch(app);
 }
 
-fn app(cx: Scope) -> Element {
-    cx.render(rsx! {
+fn app() -> Element {
+    let class = use_signal(|| String::from("content"));
+    let content = use_signal(|| String::from(include_str!("../README.md")));
```

Comment:
No need to wrap `include_str!("../README.md")` in a signal we you change the ReadOnlySignal thing